charts/backstage/ci/probes-values.yaml

```diff
@@ -1,4 +1,17 @@
 backstage:
+  appConfig:
+    techdocs:
+      builder: "local"
+    app:
+      # Let's test that everything is fine with comments
+      title: The very best Backstage Helm Chart! :D
+      baseUrl: https://somedomain.tl
+    backend:
+      baseUrl: https://somedomain.tl
+      auth:
+        keys:
+          # Is just a random b64 string for test purposes
+          - secret: N4+P+zJrErWRUGm0ZbVVhX1ZC81aSP81
```

Comment:
FYI this won't be required anymore from the next version of Backstage.
Of course let's keep it for now to get things unblocked